src/main/java/de/tum/cit/aet/artemis/communication/service/FaqService.java

```diff
@@ -0,0 +1,69 @@
+package de.tum.cit.aet.artemis.communication.service;
+
+import static de.tum.cit.aet.artemis.core.config.Constants.PROFILE_CORE;
+
+import java.util.Optional;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import de.tum.cit.aet.artemis.communication.domain.Faq;
+import de.tum.cit.aet.artemis.communication.domain.FaqState;
+import de.tum.cit.aet.artemis.communication.repository.FaqRepository;
+import de.tum.cit.aet.artemis.core.service.ProfileService;
+import de.tum.cit.aet.artemis.iris.service.pyris.PyrisWebhookService;
+
+@Profile(PROFILE_CORE)
+@Service
+public class FaqService {
```

Comment:
currently, this service only methods using the `PyrisWebhookService`. Would it make sense to use `PROFILE_IRIS` (used for both iris and pyris)?

Might not make sense if you plan to extend this service with functionality that does not rely on pyris.

Reply:
You are right in this point, ill adjust ist accordingly